app/Services/Credit/CreateInvitations.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace App\Services\Credit;
+
+use App\CreditInvitation;
+use App\Factory\CreditInvitationFactory;
+
+class CreateInvitations
+{
+
+    public function __construct()
+    {
+    }
+
+    public function __invoke($credit)
+    {
+
+        $contacts = $credit->customer->contacts;
+
+        $contacts->each(function ($contact) use($credit){
+            $invitation = CreditInvitation::whereAccountId($credit->account_id)
```

Comment:
should this be CompanyId ?

Reply:
yes sorry the entities have slightly different names in our systems (account in mine is company in yours). This is one of the reasons why its not yet ready to merge.